pom.xml

```diff
@@ -330,6 +330,27 @@ under the License.
 				</executions>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>test</id>
+						<phase>test</phase>
```

Comment:
There is no need to attach it to the test phase, if you think this scares of contributions it can be moved to mvn verify and adapt the travis script, so it will show up in the pull request.

Thinking about it, this is maybe the way to go.